### ISSUE TITLE
Updating from 1.1.0 to 1.1.1 breaks token_response.raw_id_token

### ISSUE THREAD

**Issue by chamoda** [OpenIDC/pyoidc#710]
Maybe it's my fault I may have been using a private attribute but PR https://github.com/OpenIDC/pyoidc/pull/708 removes raw_id_token.

Raw id token now accessible with `token_resp["id_token"].jwt`

**Comment by tpazderka** [comment]
If I understand it correctly, then now it should be accessible directly under `token_resp["id_token"]`.

**Comment by devchamoda** [comment]
Yes now it's only accessible with `token_resp["id_token"].jwt`. If someone have been using `token_resp.raw_id_token` it could break their production system by this minor version update.

**Comment by chamoda** [comment]
Yes now it's only accessible with token_resp["id_token"].jwt. If someone have been using token_resp.raw_id_token it could break their production system by this minor version update.

**Comment by tpazderka** [comment]
Ah, I (think I) understand it now. The `raw_id_token` was never an official part of the message and thus it has been removed.

Sorry for any inconveniences.
